finalPrep/annotations.js

```diff
@@ -2,22 +2,33 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+// Stablish a function called buildABear with a set of parameters.
 function buildABear(name, age, fur, clothes, specialPower) {
+// Create a variable called greeting and added value with an a concatenation.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`
+// Create a variable called demographics and added array.
   var demographics = [name, age]
+// Create a variable called powerSaying and add value with a concatenation.
   var powerSaying = "Did you know that I can " + specialPower + " ?"
+// Stablishin a Method
   var builtBear = {
+// cCeate a new variable conected with a previous value.
     basicInfo: demographics,
+// Create a new variable inside the object.
     clothes: clothes,
+// Create a new variable inside the object.
     exterior: fur,
+// Create a new variable inside the object.
     cost: 49.99,
+// Create a new variable inside the object.
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+// Create a new variable inside the object.
     isCuddly: true,
   }
-
+// Using a method to call a builtBear
```

Comment:
This is actually not calling the `builtBear`, but **returning** the `builtBear` object out of the function.